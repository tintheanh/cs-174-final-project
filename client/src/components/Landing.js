import React from 'react'

const Landing = () =>  
  <div className="container">
    <h1 className="text-center">WELCOME</h1>
  </div>


export default Landing
