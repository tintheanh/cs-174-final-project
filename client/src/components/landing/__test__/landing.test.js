import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Landing from '../landing';

afterEach(cleanup);

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<Landing />, div);
});

it('renders Landing correctly', () => {
	const { getByTestId } = render(<Landing />);
	expect(getByTestId('landing')).toHaveTextContent('Welcome guest');
});

it('matches snapshot', () => {
	const tree = renderer.create(<Landing />).toJSON();
	expect(tree).toMatchSnapshot();
});
