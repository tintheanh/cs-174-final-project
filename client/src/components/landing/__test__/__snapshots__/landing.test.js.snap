// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
  data-testid="landing"
>
  <h1
    className="text-center"
  >
    Welcome guest
  </h1>
</div>
`;
